apps/prairielearn/assets/scripts/lib/editAssessmentQuestionsTable.ts

```diff
@@ -0,0 +1,374 @@
+import { html } from '@prairielearn/html';
+
+import { TagBadgeList } from '../../../src/components/TagBadge.html.js';
+import { TopicBadge } from '../../../src/components/TopicBadge.html.js';
+
+function maxPoints({
+  max_auto_points,
+  max_manual_points,
+  points_list,
+  init_points,
+  assessmentType,
+}: {
+  max_auto_points: number;
+  max_manual_points: number;
+  points_list: number[];
+  init_points: number;
+  assessmentType: string;
+}) {
+  if (max_auto_points || !max_manual_points) {
+    if (assessmentType === 'Exam') {
+      return (points_list || [max_manual_points]).map((p) => p - max_manual_points).join(',');
+    }
+    if (assessmentType === 'Homework') {
+      return `${init_points - max_manual_points}/${max_auto_points}`;
+    }
+  } else {
+    return html`&mdash;`;
+  }
+}
+
+export function EditAssessmentQuestionsTable({
+  zones,
+  assessmentType,
+  showAdvanceScorePercCol,
+}: {
+  zones: any[];
+  assessmentType: string;
+  showAdvanceScorePercCol: boolean;
+}) {
+  // TO DO: Add advance score percentage column
```

Comment:
Very small nit, but this is conventional:

```suggestion
  // TODO: Add advance score percentage column
```

Reply:
Updated to `TODO` in [3c6791f](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/3c6791f5b7ef00282d313997acf9af8418da3660).